docs/conf.py

```diff
@@ -127,6 +127,10 @@ def get_version(*module_dir_components):
 # The name for this set of Sphinx documents.  If None, it defaults to
 # "<project> v<release> documentation".
 #html_title = None
+if 'READTHEDOCS_VERSION' in os.environ:
+    # Use the readthedocs version string in preference to our known version.
+    html_title = u'{} {} documentation'.format(
```

Comment:
Just a small note: I prefer to use double quotes for "human-readable text", keeping single quotes for internal identifiers.